src/components/status/index.tsx

```diff
@@ -50,11 +50,29 @@ interface IStatus {
   achievement: number;
 }
 
+interface IDB {
+  acceptor: string;
+  blockchain_url: string;
+  end_time: string;
+  executor: string;
+  price: number;
+  rate: number;
+  start_time: string;
+  status: string;
+  uuid: string;
+  volume: number;
+}
+
 const Status: React.FC = () => {
   const [listInfo, setListInfo] = useState<IListInfo[]>([]);
+  const [listInfoDB, setListInfoDB] = useState<IDB[]>([]);
   const [trainInfo, setTrainInfo] = useState<ITrainInfo[]>([]);
   const [nowIndex, setNowIndex] = useState<number>(-1);
   const [statusInfo, setStatusInfo] = useState<IStatus[]>([]);
+  const [listStop, setListStop] = useState<number>(0);
+  const [trainStop, setTrainStop] = useState<number>(0);
+  const [percentStop, setPercentStop] = useState<number>(0);
```

Comment:
1. The name of parameters `...Stop` are not clear enough. Plz change them.
2. If the value of parameter is either 0 or 1, it's better to use boolean type.

Reply:
done

Reply:
Still not clear.